content/docs/user-guide/components/reference/atom/grid.md

```diff
@@ -5,16 +5,46 @@ description: 'Open 3D Engine (O3DE) Grid component reference.'
 toc: true
 ---
 
-The **Grid** component adds a grid to the scene. You can customize the appearance of the grid, such as its colors, spacing, and size.
-
+The **Grid** component renders a wireframe grid. It is primarily used in scene-building workflows to set a frame of reference and visualize relative sizes and distances between objects.
 
 ## Provider ##
 
 [Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
 
 
-## Properties
+## Grid Component Properties
+The Grid component has simple properties for changing its size and appearance.
+
+![grid-component-base-properties](/images/user-guide/components/reference/atom/grid/grid-base-properties-ui.png)
+
+| Property | Description | Values | Default |
+|-|-|-|-|
+| **Grid Size** | The total grid size in meters. | 0.0 - 1,000,000 | 32.0 |
+| **Axis Color** | The color used for drawing grid lines for the main axis. | (0-255, 0-255, 0-255) | (0, 0, 255) |
+| **Primary Grid Spacing** | The distance between grid lines drawn in the primary color. | 0.1 to Infinity | 1.0 |
+| **Primary Color** | The color used for drawing primary grid lines. | (0-255, 0-255, 0-255) | (64, 64, 64) |
+| **Secondary Grid Spacing** | The distance between grid lines drawn in the secondary color. | 0.1 to Infinity | 0.25 |
+| **Secondary Color** | The color used for drawing secondary grid lines. | (0-255, 0-255, 0-255) | (128,128,128) |
+
+## GridComponentRequestBus
+
+| Method Name | Description | Parameter | Return | Scriptable |
```

Comment:
I missed this edit the first time, the method names should use backticks \` so that they are formatted as inline code.